src/test/java/net/snowflake/client/jdbc/FewTypesSqlData.java

```diff
@@ -0,0 +1,55 @@
+package net.snowflake.client.jdbc;
+
+import java.sql.SQLData;
+import java.sql.SQLException;
+import java.sql.SQLInput;
+import java.sql.SQLOutput;
+
+public class FewTypesSqlData implements SQLData {
```

Comment:
perhaps we can create a package in tests called `structuredtypes.sqldata` and put this class there (and SimpleClass, AllTypesClass)

Reply:
Ok